scripts/run_general_tests.py

```diff
@@ -60,6 +61,20 @@ class TestMetaInfo(font_tests.TestMetaInfo):
     expected_os2_achVendID = 'GOOG'
 
 
+class TestNames(font_tests.TestNames):
+    """Bugs:
+    https://github.com/google/roboto/issues/37
+    """
+
+    loaded_fonts = FONTS
+    family_name = 'Roboto'
+    mark_heavier_as_bold = False
+    expected_copyright = 'Font data copyright Google 2015'
```

Comment:
That is not the expected copyright line. It should be:
Copyright 2011 Google Inc. All Rights Reserved.

Where the year is the first year Roboto was released.